extensions/api/v1alpha1/sandboxtemplate_types.go

```diff
@@ -12,32 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/*
-Copyright 2025.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
```

Comment:
not sure why we had two copyright statements, so i went ahead and deleted this one